docs/illustration-guidelines.md

```diff
@@ -0,0 +1,56 @@
+# Primer Illustration Notes
+
+Guidelines for creating Illustrations for Primer
+
+# Symbols
+
+Primer illustrations are made up of a collection of various symbols which will be referred to as illustration components in order to maintain consistency between documentation and Figma. These components are meant to come together and represent the connection between Design + Code within the system. 
+
+This is a list of the current symbols used in Primer illustrations
+
+- Paintbrush
+- Squiggles
+- Dots
+- Geometric Shapes
+- Code Lines
+
+## Paintbrush
+
+The paintbrush is used to represent the design/artistic side of Primer. It should only appear once per illustration.
+
+## Squiggles
+
+Squiggles come in two styles: 2px stroke/`blue-400` and 1px stroke/`blue-200`. They can vary in size but are typically `40x8` or `20x4`. Squiggles should typically only be horizontal or vertical, but if they are angled please use a 45° angle. 
+
+## Dots
+
+Dot patterns can be used to add some contrast and texture beneath elements. Currently we are only using the uniform dot grid as a drop shadow in illustrations. On dark backgrounds use `blue-700` and on light backgrounds `blue-200`.
+
+## Geometric Shapes
+
+Primer illustrations use 4 types of geometric shapes. These shapes are:
+
+- Circle
+- Square*
+- Triangle
+- Hexagon
+
+*The square shape can be rotated 45° to register as a diamond as well.
+
+Shapes should only have a stroke and no fill. The stroke styles are `2px stroke/blue-400` and `1px stroke/blue-200`. When placing shapes next to one another the styling should be different.
+
+## Code Lines
+
+Code lines are represented by a row of rectangular blocks.
+
+**Dimensions**
+
+- Block width should be at a  `base-8` scale.  Block height should be limited to  `16px`, `24px` or `32px`.  (THESE ARE NOT DEFINITE RIGHT NOW)
+
+**Spacing**
+
+- The space between blocks should be on a `base-8` scale and have the same amount of space between on both sides. Please refer to [Primer: Spacing guidelines](https://styleguide.github.com/primer/support/spacing/) for more information. (THESE ARE NOT DEFINITE RIGHT NOW)
+
+# Spacing
```

Comment:
There are two spacing sections right next to each other, confused what the second one is and what `base of 2` is about since we use a base 8 scale.

Reply:
@broccolini Whoops! That is not supposed to be there. The first section is correct.